cdap-docs/faqs/source/cloudera-manager.rst

```diff
@@ -66,16 +66,16 @@ not have direct network access), the parcels are available by their full URLs. A
 stored in a directory that does not offer browsing, they are listed here:
 
 .. parsed-literal::
-  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-el6.parcel
-  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-precise.parcel
-  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-trusty.parcel
-  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-wheezy.parcel
+  |http:|//repository.cask.co/parcels/cdap/|short-version|/CDAP-|version|-1-el6.parcel
```

Comment:
we will have an -el7.parcel now too, since https://github.com/caskdata/cdap/pull/5119